uniprotkb-common/src/test/java/org/uniprot/api/uniprotkb/common/service/uniprotkb/UniProtEntryQueryResultsConverterTest.java

```diff
@@ -71,12 +74,14 @@ void canConvertInactiveDeletedDocToEntry() {
         EntryInactiveReason inactiveReason = entry.getInactiveReason();
         assertEquals(InactiveReasonType.DELETED, inactiveReason.getInactiveReasonType());
         assertEquals(DeletedReason.SOURCE_DELETION_EMBL, inactiveReason.getDeletedReason());
+        assertEquals(uniParcDeleted, entry.getExtraAttributeValue(UNIPARC_ID_ATTRIB));
     }
 
```

Comment:
Not applicable to this task.. but I see no tests for 'convertQueryResult' method.
also no check for add lineage when tax entry does not exist